salt/states/rsync.py

```diff
@@ -97,7 +99,10 @@ def synchronized(name, source,
     '''
     ret = {'name': name, 'changes': {}, 'result': True, 'comment': ''}
 
-    if not os.path.exists(source):
+    if name.strip() == '/':
```

Comment:
I'm really not so sure about this. I don't like Salt being quite so opinionated about what's "safe" in this case. Can you please explain your reasoning further?